### PR TITLE
fixing sign-conversion warnings, part 17, utp_stream

### PR DESCRIPTION
**Pull request by aldenml**

### CODE REVIEW

File: src/utp_stream.cpp [hunk 1]
```diff
@@ -1714,7 +1714,7 @@ bool utp_socket_impl::send_pkt(int const flags)
 	// congestion window and the advertised receive window from
 	// the other end.
 	if (m_bytes_in_flight + payload_size > std::min(int(m_cwnd >> 16)
-		, int(m_adv_wnd - m_bytes_in_flight)))
+		, aux::numeric_cast<int>(m_adv_wnd) - m_bytes_in_flight))
```
Comment (aldenml) [inline src/utp_stream.cpp]: I need careful review here

File: src/utp_stream.cpp [hunk 2]
```diff
@@ -1904,8 +1904,8 @@ bool utp_socket_impl::send_pkt(int const flags)
 	}
 
 	h->timestamp_difference_microseconds = m_reply_micro;
-	h->wnd_size = std::max(m_in_buf_size - m_buffered_incoming_bytes
-		- m_receive_buffer_size, std::int32_t(0));
+	h->wnd_size = aux::numeric_cast<std::uint32_t>(std::max(m_in_buf_size - m_buffered_incoming_bytes
```
Comment (aldenml) [inline src/utp_stream.cpp]: and here

**Comment by aldenml** [comment]
if you have a minute, to restart the macOS travis CI job, thanks

**Comment by arvidn** [comment]
lgtm

**Pull request merged**